translations/compfest-11-penyisihan/compfest-11-penyisihan-membangun-pos-keamanan.html

```diff
@@ -0,0 +1,60 @@
+<!-- Title : Building Security Post -->
+
+<h3> Description </h3>
+
+<p>
+	Chanek is a country consisting of $N$ cities and $M$ roads. Uniquely, every city in Chanek is connected with $\textbf{exactly two}$ other cities.
+</p>
+
+<p>
+	Mr. Chanek, the president of Chanek, wanted to build as many security posts as possible. A post will be located in one of the city in Chanek. These posts will guard the roads connected to the city where the post is built. However, Mr. Chanek do not want any road to be guarded by two posts as this may cause dispute between the two posts.
```

Comment:
```suggestion
	Mr. Chanek, the president of Chanek, wanted to build as many security posts as possible. A post will be located in one of the city in Chanek. This post will guard the roads connected to the city where the post is built. However, Mr. Chanek do not want any road to be guarded by two posts as this may cause dispute between the two posts.
```